event/event.gen.go

```diff
@@ -545,63 +583,63 @@ func Playing(listener func(*vecty.Event)) *vecty.EventListener {
 	return &vecty.EventListener{Name: "playing", Listener: listener}
 }
 
-// The pointer was locked or released.
-// https://developer.mozilla.org/docs/Web/Events/pointerlockchange
-func PointerLockChange(listener func(*vecty.Event)) *vecty.EventListener {
-	return &vecty.EventListener{Name: "pointerlockchange", Listener: listener}
-}
-
-// It was impossible to lock the pointer for technical reasons or because the permission was denied.
-// https://developer.mozilla.org/docs/Web/Events/pointerlockerror
-func PointerLockError(listener func(*vecty.Event)) *vecty.EventListener {
-	return &vecty.EventListener{Name: "pointerlockerror", Listener: listener}
-}
```

Comment:
They've moved to [there](https://github.com/gopherjs/vecty/pull/11#discussion_r52086811).